frontend/src/components/StreamingBox.vue

```diff
@@ -90,14 +100,20 @@
 </template>
 
 <script>
-import get from 'lodash/get'
 import ifvisible from 'ifvisible'
-
 import Janus from '@src/lib/janus'
-import ViewingThrottle from '@src/lib/viewing_throttle'
+import { StreamThrottle, STREAM_SESSION, STREAM_EVENT } from '@src/lib/stream-throttle'
+import { isLocalStorageSupported } from '@static/js/utils'
+
+const STREAM_STATUS = {
+  Unavailable: 'Unavailable',
+  Available: 'Available',
+  Started: 'Started',
+}
```

Comment:
Main refactoring idea is here. Previously we had `isVideoAvailable` and `isVideoVisible` variables, defining the state of the video. But they doesn't describe it clearly (in my opinion).

Now we have 3 main states of the video:
1. Unavailable
2. Available – webrtc steam is ready and can be attached to the <video /> tag
3. Started – stream is attached to the <video /> and will be played right after downloading enough part

At the 3rd step video could be in loading status (`isVideoLoading`) or in buffering (`isTrackMuted`)